docs/tutorials/ifci_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,95 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+********************************************
+Exporting IFCI Logs Using QGroundControl
+********************************************
+
+IFCI data can be found in the flight controller logs. These logs can be exported from the flight controller using QGroundControl.
```

Comment:
Maybe change it to "Motor telemetry data can be found...". At least in PX4 this data is collected by DroneCAN, DShot, and IFCI, then stored in the same data structure.